std_msgs/Header header
quad_ufabc/CartesianPoint cartesian_point 
